charts/rstudio-connect/templates/_helpers.tpl

```diff
@@ -89,7 +89,9 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 */}}
 {{- define "rstudio-connect.runtimeYaml" -}}
   {{- $runtimeYaml := .Values.launcher.customRuntimeYaml }}
-  {{- if $runtimeYaml }}
+  {{- if eq $runtimeYaml "pro" }}
+    {{- .Files.Get "default-runtime-pro.yaml" }}
+  {{- else if $runtimeYaml }}
```

Comment:
I'm confused by the code compared to the documentation. This field is not really a Boolean, as documented. Here, we see that it might be a string-with-meaning (pro), or a string-with-meaning (YAML-as-string), or a YAML object, or some-value-that-is-not-truthy (default YAML).

All of this complexity is not really captured in the documentation.

What about:

`customRuntimeYaml` is an object allowing values such as:

* When not specified, use the base images (default).
* When falsy, use the base images (backwards-compatible).
* When "base", use the base images.
* When "pro", use the pro images.
* When a string, interpret as YAML.
* When an object, assume well-structured YAML.

It would also be handy to link to the "base" and "pro" YAML files as examples.

We could probably expand this field to take list values, but we can postpone that.

Reply:
I like that idea! Well said!